estimagic/differentiation/differentiation_auxiliary.py

```diff
@@ -1,68 +1,63 @@
 """Auxiliary functions for a one step differentiation."""
 
 
-def central(func, f_x0, params, var, h, *func_args, **func_kwargs):
+def central(internal_func, f_x0, params_value, i, h):
     """Calculate the central difference.
 
     Args:
-        func (func): A function that maps params_sr into a float.
-        f_x0 (func output): The function value at params_sr, func_args and func_kwargs.
-        params (DataFrame): see :ref:`parmas_df`
-        var (string): The variable of which the finite difference is calculated.
+        internal_func (func): A function wrapping the original function, s.t. it is
```

Comment:
The docstring should not explain that `internal_func` it is a wrapper around some other function. It should only explain which type of function is a valid input and which isn't. Being wrapped is not a requirement, it is only necessary, that the function takes just one argument and this argument is a 1d numpy array. This probably applies at several places.

Reply:
Changed description to: A function with a 1d numpy array as it's only argument.